packages/admin/admin/src/form/ChipSelect.tsx

```diff
@@ -0,0 +1,201 @@
+
+export type ChipSelectClassKey = "root" | "inputBase" | "inputRoot" | "chip" | "select";
+
+type OwnerState = {
+    fullWidth: boolean;
+};
+
+const Root = createComponentSlot("div")<ChipSelectClassKey, OwnerState>({
+    componentName: "ChipSelect",
+    slotName: "root",
+})(
+    ({ ownerState }) => css`
+        ${!ownerState.fullWidth &&
+        css`
+            width: fit-content;
+        `}
+    `,
+);
+
+const ChipInputRoot = createComponentSlot("div")<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "inputRoot",
+})(
+    () => css`
+        display: flex;
+        flex-direction: column;
+        position: relative;
+    `,
+);
+
+const Chip = createComponentSlot(MuiChip)<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "chip",
+})(
+    () =>
+        css`
+            justify-content: flex-start;
+        `,
+);
+
+const InputBase = createComponentSlot(MuiInputBase)<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "inputBase",
+})(
+    () => css`
+        position: absolute;
+        inset: 0;
+        opacity: 0;
+
+        .MuiInputBase-input {
+            padding: 0;
+            height: 100% !important;
+            min-height: unset !important;
+        }
+    `,
+);
+
+const LabelInputBase = createComponentSlot(MuiInputBase)<ChipSelectClassKey>({
+    componentName: "ChipSelect",
+    slotName: "inputBase",
+})(
+    () => css`
+        all: unset;
+        padding-right: 0 !important;
+
+        > div {
+            padding-right: 0 !important;
+        }
+
+        .MuiInputBase-input {
+            padding: 0;
+            height: 100% !important;
+            min-height: unset !important;
+        }
+    `,
+);
+
+export interface ChipSelectProps<T = string>
+    extends ThemedComponentBaseProps<{
+            root: "div";
+            inputRoot: "div";
+            select: typeof Select;
+            menuItem: typeof MenuItem;
+            inputBase: typeof MuiInputBase;
+        }>,
+        Omit<ChipProps, "children" | "onChange"> {
+    children?: React.ReactNode;
+    getOptionLabel?: (option: T) => string;
+    getOptionValue?: (option: T) => string;
+    options?: T[];
+    selectedOption?: T;
+    onChange: (event: SelectChangeEvent) => void;
+    fullWidth?: boolean;
+}
+
+type ChipInputProps = InputBaseProps & {
+    chipProps?: Omit<ChipProps, "children">;
+    inputBaseProps?: InputBaseProps;
+    inputRootProps?: React.ComponentPropsWithoutRef<"div">;
+};
+
+const ChipInput = ({ chipProps, inputBaseProps, inputRootProps, ...restProps }: ChipInputProps) => (
+    <ChipInputRoot {...inputRootProps}>
+        <Chip
+            icon={<ChevronDown />}
+            label={<LabelInputBase {...inputBaseProps} {...restProps} />} // use second InputBase to show the label
+            variant="filled"
+            {...chipProps}
+        />
+        <InputBase {...inputBaseProps} {...restProps} />
```

Comment:
I don't think this second `InputBase`, outside of the label is necessary. 
If the props are passed to the `LabelInputBase` correctly and the styling spans it over the entire `Chip` it should work.

Reply:
No it doesn't since the label has a container where the InputBase goes. The container is not full width and therefore not clickable. `position: absolute` also doesn't work here.